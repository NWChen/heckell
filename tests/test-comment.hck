/*
  multi
    line
      comment
*/
let x : int;
x  = 2;	    // this will now show up in output

// nor /* will this

let s : string;
s = "hello \(3 /* cant see me */ + 3)";
