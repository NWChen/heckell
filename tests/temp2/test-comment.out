let x: int
x = 2
let s: string
s = "hello \( (3 + 3) )"
